modules/wayland/xkbMacKeyboardConfig

```diff
@@ -0,0 +1,5 @@
+XKB_DEFAULT_VARIANT=mac
+XKB_DEFAULT_MODEL=apple_laptop
+# lvl3 is the third level of the keyboard, which is used for keys like | {} and []
```

Comment:
The given exmamples for the third level (Shift + Alt + 7) are for a German keyboard, no? In case you wanted to include an English example `°` (Shift + Alt + 8) would be one.